LibCarla/source/carla/ros2/ROS2.cpp

```diff
@@ -1062,27 +1055,27 @@ void ROS2::ProcessDataFromStatusSensor(
   constexpr uint8_t hazard_lights_mask = 0b00000100;
 
   // Decode data
-  const bool is_reverse = control_flags & reverse_mask;
-  const bool is_manual_gear = control_flags & manual_gear_mask;
+  const bool is_reverse = data.control_flags & reverse_mask;
+  const bool is_manual_gear = data.control_flags & manual_gear_mask;
 
-  const bool is_left_blinker_on = turn_mask & left_blinker_mask;
-  const bool is_right_blinker_on = turn_mask & right_blinker_mask;
-  const bool is_hazard_lights_on = turn_mask & hazard_lights_mask;
+  const bool is_left_blinker_on = data.turn_mask & left_blinker_mask;
+  const bool is_right_blinker_on = data.turn_mask & right_blinker_mask;
+  const bool is_hazard_lights_on = data.turn_mask & hazard_lights_mask;
 
   // TODO: Verify whether any of the fields should be inverted
   // Now all positive values mean forward and left (if going forward)
-  _autoware_publisher->SetVelocity(vel_x_mps, -vel_y_mps, -angVel_z_mps);
+  _autoware_publisher->SetVelocity(data.vel_x_mps, -data.vel_y_mps, -data.angVel_z_mps);
 
-  // TODO: Check if steering should be set reversed (it is set reversed because control had to be reversed (this is an educated guess))
-  _autoware_publisher->SetSteering(-steer);
+  // TODO: Check if data.steering should be set reversed (it is set reversed because control had to be reversed (this is an educated guess))
```

Comment:
```suggestion
  // TODO: Check if steering should be set reversed (it is set reversed because control had to be reversed (this is an educated guess))
```
I think this was a mistake in renaming.

Reply:
Yes it was, I've didn't notice it changed all keywoard `steering`.